kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -670,43 +674,43 @@ def compute_jumpdests(sections: list[KInner]) -> KInner:
     :return: This function iterates over each section, appending the jump destinations (0x5B) from the bytecode in a KAst Set.
     :rtype: KInner
     """
-    offset = 0
-    jumpdests = []
+    mutable_jumpdests = bytearray(b'')
     for s in sections:
         if type(s) is KApply and s.label == KLabel('buf'):
             width_token = s.args[0]
             assert type(width_token) is KToken
-            offset += int(width_token.token)
+            mutable_jumpdests += bytes(int(width_token.token))
         elif type(s) is KToken and s.sort == BYTES:
             bytecode = pretty_bytes(s)
-            jumpdests.extend(_process_jumpdests(bytecode, offset))
-            offset += len(bytecode)
+            mutable_jumpdests += _process_jumpdests(bytecode)
+            len(bytecode)
         else:
             raise ValueError(f'Cannot compute jumpdests for type: {type(s)}')
 
-    return set_of(jumpdests)
+    return token(bytes(mutable_jumpdests))
 
 
-def _process_jumpdests(bytecode: bytes, offset: int) -> list[KToken]:
+def _process_jumpdests(bytecode: bytes) -> bytes:
     """Computes the location of JUMPDEST opcodes from a given bytecode while avoiding bytes from within the PUSH opcodes.
 
     :param bytecode: The bytecode of the contract as bytes.
     :type bytecode: bytes
     :param offset: The offset to add to each position index to align it with the broader code structure.
     :type offset: int
-    :return:  A list of intToken instances representing the positions of all found jump destinations in the bytecode adjusted by the offset.
-    :rtype: list[KToken]
+    :return: A bytes object with the same length as the input of zeros. Valid jumpdests are represented by the value 1.
```

Comment:
I'm not sure, what is "input of zeros" here? Isn't it same length than bytecode with the zero input, or something like that?

Reply:
Updated.